src/Resources/contao/drivers/DC_Folder.php

```diff
@@ -172,6 +178,8 @@ public function __construct($strTable)
 		$this->strTable = $strTable;
 		$this->blnIsDbAssisted = $GLOBALS['TL_DCA'][$strTable]['config']['databaseAssisted'];
 
+		$this->initPicker();
```

Comment:
This line needs to be removed entirely, because the picker will be initialized 10 lines below. I must have missed this during rebase.